### DISCUSSION TITLE
Prisma Docs Generator

### DISCUSSION THREAD

**Discussion by pantharshit00** [prisma/prisma#3201] [Q&A]
Just wanted to post this here as well since we have quite a lot of watchers in this repository.

I made a side project which lets you generate a docs reference for your Prisma schema: https://github.com/pantharshit00/prisma-docs-generator

![image](https://user-images.githubusercontent.com/22195362/89279770-630c1a80-d665-11ea-9074-183d1bc75d00.png)

Check it out if you are interested and star the repo if you like it. Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
